functional-test/src/test/groovy/org/rundeck/tests/functional/api/job/JobActivityHistorySpec.groovy

```diff
@@ -0,0 +1,54 @@
+package org.rundeck.tests.functional.api.job
+
+import org.rundeck.util.annotations.APITest
+import org.rundeck.util.container.BaseContainer
+
+@APITest
+class JobActivityHistorySpec extends BaseContainer {
+    def setupSpec() {
+        startEnvironment()
+
+    }
+
+    def "Retrieve job Activity history from Job List Page "() {
+        given: "A Job exists in the project with execution history"
+        def jobId = "3957f735-5269-4c85-bc57-f123598886f6"
+
+        when: "The activity history for the job is requested"
+
+        def response = doGet("/job/${jobId}/executions")
```

Comment:
Use org.rundeck.util.common.jobs.JobUtils functions to retrieve job data. This is the layer where the REST API calls are meant to be isolated. Avoid calling the API directly from the tests.